now.json

```diff
@@ -0,0 +1,11 @@
+{
+    "build": {
+        "env": {
+            "REACT_APP_GOOGLE_TRACKING_ID": "@UA-150653982-1",
+            "REACT_APP_AUDIENCE": "@https://sauti-africa-market-prices.auth0.com/api/v2/",
+            "REACT_APP_DOMAIN": "@sauti-africa-market-prices.auth0.com",
+            "REACT_APP_CLIENT_ID": "@bxosYDeCRsMMIEuRFyte2kxifC5uK9ua",
+            "REACT_APP_HOST": "@http://localhost:3000"
```

Comment:
should this still be local host?

Reply:
I don't think we need this file anymore, I believe it was the config file for zeit